config.yaml

```diff
@@ -27,6 +27,24 @@ source_config:
 # remove to disable S3 uploads
 s3_bucket: fcos-builds
 
+default_artifacts:
+  x86_64:
+        - Aliyun
+        - AWS
+        - Azure
+        - AzureStack
+        - DigitalOcean
+        - Exoscale
+        - GCP
+        - IBMCloud
+        - Live
+        - Nutanix
+        - Metal
+        - Metal4k
+        - OpenStack
+        - VirtualBox
+        - VMware
+        - Vultr
```

Comment:
Minor: I still think it'd be more appropriate in this config file to have the artifact names be lowercase. And ideally we also stop lowercasing in the code too.

We're defining a new config format, which may live outside this repo in the future, so I'd lean towards starting strict and simple and loosening later on if we really perceive a need.

This also makes the deduplication mentioned earlier safer.